build-external-api-jar/tests/src/test/scala/org/virtuslab/tests/OpenProjectAndBuildArtifacts.scala

```diff
@@ -0,0 +1,166 @@
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.naming.ldap" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.naming.rmi" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.net" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.pack" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.rmic" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.scripting.nashorn" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.scripting.nashorn.shell" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.sctp" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.security.auth" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.security.jgss" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.unsupported" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.unsupported.desktop" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.xml.dom" type="simple" />
+      |            <root url="jrt:///usr/lib/jvm/java-11-openjdk-amd64!/jdk.zipfs" type="simple" />
+      |          </root>
+      |        </classPath>
+      |        <javadocPath>
+      |          <root type="composite">
+      |            <root url="https://docs.oracle.com/en/java/javase/11/docs/api/" type="simple" />
+      |          </root>
+      |        </javadocPath>
+      |        <sourcePath>
+      |          <root type="composite" />
+      |        </sourcePath>
+      |      </roots>
+      |      <additional />
+      |    </jdk>
+      |  </component>
+      |</application>
+      """.stripMargin
+  }
+   val artifactVal =
+      """
+        |<component name="ArtifactManager">
+        |  <artifact type="jar" name="external-system-test-api">
+        |    <output-path>$PROJECT_DIR$/out/artifacts/external_system_test_api</output-path>
+        |    <root id="archive" name="external-system-test-api.jar">
+        |      <element id="module-test-output" name="intellij.platform.externalSystem.impl" />
+        |      <element id="module-test-output" name="intellij.platform.externalSystem.tests" />
+        |    </root>
+        |  </artifact>
+        |
+        |
+        |</component>""".stripMargin
+}
+
+
+class OpenProjectAndBuildArtifacts extends IdeProbeTestSuite with ProbeExtensions {
+
+  import OpenProjectAndBuildArtifacts._
+
+  registerFixtureTransformer(
+    _.withAfterIntelliJInstall{ (_, installedIntelliJ) =>
+      installedIntelliJ.paths.config.resolve( "options/jdk.table.xml" ).write(OpenProjectAndBuildArtifacts.jdkDef)
```

Comment:
Is it possible to keep these XML files as test resources? Then you could just copy them to their destination using java.nio methods or even a shell script.

Reply:
I guess this may slightly change when we'll be going to build artifacts for 2021.1, so I'd wait until it's released